_schema.graphql

```diff
@@ -470,6 +470,12 @@ type AnalyticsShow {
   entityId: String!
 }
 
+# Statistics for users
+type AnalyticsUserStats {
+  numberOfPurchases: Int!
```

Comment:
Just to call out if we like the name of this field before we move this PR forward.

`numberOfPurchases` in Vortex is the count that is getting returned from Vortex. @sepans made a good point that now is a good time to see if we like the name - before we merge the change into metaphysics

Reply:
**suggestion (non-blocking):** `purchaseCount` or `totalPurchaseCount` seems to be more consistent with current convention.

*format via [Conventional Comments](https://conventionalcomments.org/) + [saved replies](https://github.com/settings/replies)*

Reply:
`totalPurchaseCount` sounds like a good idea! Adjusting that here and also in Vortex [here](https://github.com/artsy/vortex/pull/265)